joomla/joomla-cms
# [4.2] Display one set of buttons

Pull Request for Issue #39646 .

### Summary of Changes

When @laoneo did the PR for tinyMCE https://github.com/joomla/joomla-cms/pull/39625 I've contacted him with this request that was left for another PR

![Screenshot 2023-01-16 at 16 53 32](https://user-images.githubusercontent.com/3889375/212719386-f17f0784-77c3-4086-8354-ffad844a25af.png)

### Testing Instructions

add these lines after https://github.com/joomla/joomla-cms/blob/4dcee6991fdbbef2daa20168d930f3b4e790dcb3/administrator/components/com_content/forms/article.xml#L58
```xml
<field name="articletext1" type="editor" editor="none" buttons="true" hide="menu,module" />
<field name="articletext2" type="editor" editor="codemirror" buttons="true" hide="article,image" />
```

add these lines after https://github.com/joomla/joomla-cms/blob/4dcee6991fdbbef2daa20168d930f3b4e790dcb3/administrator/components/com_content/tmpl/article/edit.php#L72

```
<?php echo $this->form->getInput('articletext1'); ?>
<?php echo $this->form->getInput('articletext2'); ?>
```

Create a new article, tinyMCE should have all the buttons, none should miss the menu and module and codemirror should miss article and image
### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Link to documentations
Please select:
